### ISSUE TITLE
Icon color white

### ISSUE THREAD

**Issue by giancafigueroa** [jodit/jodit-angular#34]
I have one problem with the jodit-editor the iconos somethings change de color 
![image](https://user-images.githubusercontent.com/12590012/59553745-18aa7480-8f5f-11e9-9615-4c109fc92e55.png)

**Comment by giancafigueroa** [comment]
I think, the release was bad, becose the library is updated but, when I imported from npm not changed. I saw it in the zip the release the code is older than actual.
I did fork to repository and pusblish again the version  and I fixed this problem and I can use jodit-angular with the latest version jodit.

https://www.npmjs.com/package/jodit-angular-fork

[simplescreenrecorder-2019-09-26_09.33.04.zip](https://github.com/jodit/jodit-angular/files/3658294/simplescreenrecorder-2019-09-26_09.33.04.zip)
 @xdan 
@jodit
